File: PASCAL/Cap06/c06ex02.pas
program c06ex02;

var
  NOTAS: array[1..8, 1..4] of Real;
  I, J: Integer;

begin
  for I := 1 to 8 do
    begin
      WriteLn('Entre as notas do aluno ', I, ':');
      for J := 1 to 4 do
      begin
        Write('Nota ', J, ': ');
        ReadLn(NOTAS[I, J]);
      end;
      WriteLn;
  end;

  WriteLn;
  WriteLn('RELATORIO DE NOTAS');
  WriteLn;
  WriteLn('Aluno Nota1 Nota2 Nota3 Nota4');
  WriteLn('----- ----- ----- ----- -----');
  for I := 1 to 8 do
    begin
      Write(I:5, ' ');
      for J := 1 to 4 do
      begin
        Write(NOTAS[I, J]:5:1, ' ');
      end;
      WriteLn;
    end;

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: C/Cap06/c06ex02.c
// c06ex02.c

#include <stdio.h>

int main(void) {
  double NOTAS[8][4];
  int I, J;

  for (I = 0; I <= 7; ++I) {
    printf("Entre as notas do aluno %d:\n", I + 1);
    for (J = 0; J <= 3; ++J) {
      printf("Nota %d: ", J + 1);
      scanf("%lf", &NOTAS[I][J]);
      while (getchar() != '\n');
    }
    printf("\n");
  }

  printf("\n");
  printf("RELATORIO DE NOTAS\n");
  printf("\n");
  printf("Aluno Nota1 Nota2 Nota3 Nota4\n");
  printf("----- ----- ----- ----- -----\n");
  for (I = 0; I <= 7; ++I) {
    printf("%5d ", I + 1);
    for (J = 0; J <= 3; ++J)
      printf("%5.1lf ", NOTAS[I][J]);
    printf("\n");
  }

  printf("\n");
  printf("Tecle <Enter> para encerrar... ");
  while (getchar() != '\n');

  return 0;
}
